site/README.md

```diff
@@ -151,6 +162,20 @@ Retrieves the user object of the currently logged in user.
 - Request Response:
   - `user`: the user currently authenticated with the API
 
+### `GET /api/me/apiKeys` [authenticated]
```

Comment:
I agree but would rather we do this in follow-up since it'll involve renaming every other API route (or having this one oddly inconsistent with the others)

Reply:
https://app.asana.com/0/1201095311341924/1201685191027589/f